storybook/stories/Documentation.mdx

```diff
@@ -0,0 +1,5 @@
+import { Canvas, Markdown, Meta } from '@storybook/blocks';
+
+<Meta title="Documentation" />
+
+Read the typedocs <a href="typedocs/index.html" target="_blank" rel="noopener">here</a>.
```

Comment:
I think with dropping extra `target` and `rel` attributes works too

Reply:
Yes